.github/workflows/test-gpu.yml

```diff
@@ -51,10 +51,17 @@ jobs:
       - name: Nvidia SMI sanity check
         run: nvidia-smi
 
+      - name: Extract max Python version from classifiers
+        run: |
+          sudo apt-get update && sudo apt-get install -y cargo && cargo install toml-cli
```

Comment:
You could also use this: https://github.com/marketplace/actions/cargo-install

Reply:
ah was looking for something like this just now.  I also thought cargo came on the ubuntu images by default but evidently not `install`? 

Also, do you think we should be doing this for all of our CI i.e., extracting from `pyproject.toml`?

Reply:
Ok I think that is only for rust projects: https://github.com/scverse/anndata/actions/runs/11330444061/job/31508293550?pr=1714

Reply:
I still think that using that action is better, because it sets up PATH and uses caching.

Reply:
https://github.com/scverse/anndata/actions/runs/11330444061/job/31508293550 Not sure

Reply:
I read this as `cargo` not being on the path